rodan-main/code/rodan/jobs/MEI_encoding/build_mei_file.py

```diff
@@ -586,15 +635,48 @@ def build_mei(pairs: List[Tuple[List[dict], dict]], classifier: dict, width_cont
 
                 continue
 
+
             cur_staff = int(glyph['staff'])
 
+            # if multi column and next system is new column, add new cb
+            if is_multi_column:
+                if staff_to_column[cur_staff] > prev_column:

```

Comment:
Combine if statements with an and instead of nesting them - lot of indentation already